inst/resources/css/jss.css

```diff
@@ -0,0 +1,268 @@
+:root {
+  --background: #2888b8;
+  --width: 210mm;
+  --color-paper: white;
+  --screen-pages-spacing: 5mm;
+  --header-width: 462px;
+  --header-image-padding: 18px;
+}
+
+html {
+  font-size: 14px;
+}
+
+/* article header */
+.document-header {
+  border-width: 4px 0;
+  border-style: solid;
+  width: var(--header-width);
+  position: relative;
+  margin-left: auto;
+  margin-bottom: 110px;
+}
+.document-header::before {
+  content: "";
+  display: block;
+  position: absolute;
+  left: calc(var(--header-width) - var(--width) + var(--margin-left) + var(--margin-right));
+  border-width: 4px 0;
+  border-style: solid;
+  border-color: transparent;
+  top: -4px;
+  height: 100%;
+  width: calc(var(--width) - var(--margin-left) - var(--margin-right) - var(--header-width) - var(--header-image-padding));
+  background-clip: border-box;
+  background-image: url(https://www.jstatsoft.org/public/site/images/jstatsoft/jsslogo.jpg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-origin: border-box;
+}
+.journal-name {
+  padding-top: 14px;
+  padding-left: 20px;
+  filter: invert(100%);
```

Comment:
I see :)

Reply:
this is so weird (it was funny to do but I definitely don't like that).